exporter/exporterhelper/README.md

```diff
@@ -1,28 +1,101 @@
 # Exporter Helper
 
-This is a helper exporter that other exporters can depend on. Today, it
-primarily offers queued retries  and resource attributes to metric labels conversion.
+This is a helper exporter that other exporters can depend on. It offers several
+independent capabilities including:
+
+- Generic export settings, like `timeout`
+- In-memory queuing
+- Ability to retry an export operation
 
 > :warning: This exporter should not be added to a service pipeline.
 
 ## Configuration
 
-The following configuration options can be modified:
+The below configuration options can be modified. The full list of settings
+exposed for this helper exporter are documented [here](factory.go).
+
+### Export Settings
+
+These settings modify the behavior when a payload is exported.
+
+- `resource_to_telemetry_conversion`
+  - `enabled` (default = `false`): If `enabled` is `true`, all the resource
+    attributes will be converted to metric labels.
+- `timeout` (default = `5s`): Time to wait per individual attempt to export data
+  to a backend. If the backend accepts the data or returns an error before the
+  configured `timeout` then the `timeout` setting is ignored. This setting is
+  completely independent of `max_elapsed_time` under `retry_on_failure.`
+
+### Queuing
+
+These settings modify in-memory queuing. It is recommended to keep queuing
+enabled so data is not lost during export in case of error or timeout.
+Note the larger the queue the more memory is required. In general,
+the default settings should not be modified.
```

Comment:
I don't agree with this. For example, having batch processor with 1s, 5000 queue elements result in keeping an hour of incoming data in memory. In most cases this is unacceptable. I don't know if this is appropriate place to mention batch processor either

Reply:
I would say it is a general case, which one can forget of